cli/internal/helm/loader_test.go

```diff
@@ -125,6 +125,76 @@ func TestTemplate(t *testing.T) {
 	}
 }
 
+// TestOperators checks if the rendered constellation-services chart produces the expected yaml files.
+func TestOperators(t *testing.T) {
+	testCases := map[string]struct {
+		csp cloudprovider.Provider
+	}{
+		"GCP": {
+			csp: cloudprovider.GCP,
+		},
+		"Azure": {
+			csp: cloudprovider.Azure,
+		},
+		"QEMU": {
+			csp: cloudprovider.QEMU,
+		},
+	}
+
+	for name, tc := range testCases {
+		t.Run(name, func(t *testing.T) {
+			assert := assert.New(t)
+			require := require.New(t)
+
+			chartLoader := ChartLoader{joinServiceImage: "joinServiceImage", kmsImage: "kmsImage", ccmImage: "ccmImage", cnmImage: "cnmImage", autoscalerImage: "autoscalerImage"}
+			release, err := chartLoader.Load(tc.csp, true, []byte("aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"), []byte("aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"), []uint32{1, 11}, false)
```

Comment:
Isn't this function already covered by `TestLoad`?
If you want to test the chart functions individually, I would suggest directly running test on  `loadCilium`, `loadConstellationServices`, and `loadOperators`.